docs/administration/data/data-migration.md

```diff
@@ -29,6 +29,16 @@ The data migration tools are not suitable for every imaginable scenario. In thes
 1. To move your entire Octopus Server from a self-hosted installation to Octopus Cloud, see [migrating from self-hosted to Octopus Cloud](/docs/octopus-cloud/migrations.md).
 1. To move your entire Octopus Cloud instance to a self-hosted Octopus Server please [contact our support team](mailto:support@octopus.com) for advice and assistance.
 
+:::problem
+**Unsupported scenarios**
+
+Sometimes using the data migration tool may look like it could solve a problem, but in fact will make things worse. Here are some scenarios we've seen that are explicitly not supported.
+
+1. **{{ Export , Modify , Import }}**
+
+Unfortunately, since the import isn't running all of the same validation checks as the API, using an `export` -> `modify` -> `import` can modify your data in such a way that is invalid for the API. Some scenarios _might work_ but because at this point you're effectively hand editing your data, it isn't something we support.
```

Comment:
slight tweak:
```suggestion
Unfortunately, since the import isn't running all of the same validation checks as the API, using an **{{ export, modify, import}}** can modify your data in such a way that is invalid for the API. Some scenarios _might work_ but because at this point you're effectively hand editing your data, this isn't something we support.
```